tests/InputFiles/FindHorizons/FindHorizons3D.yaml

```diff
@@ -0,0 +1,45 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: FindHorizons3D
+# Check: parse
+
+DomainCreator:
+  Shell:
+    InnerRadius: 1.9
+    OuterRadius: 10.
+    InitialRefinement: 0
+    InitialGridPoints: [8, 8]
+    UseEquiangularMap: True
+    EquatorialCompression: None
+    WhichWedges: All
+    RadialPartitioning: []
+    RadialDistribution: [Logarithmic]
+    TimeDependence: None
+
+Importers:
+  VolumeData:
+    FileGlob: "KerrVolume*.h5"
+    Subgroup: "VolumeData"
+    ObservationValue: Last
+
+ApparentHorizons:
+  AhA:
```

Comment:
@markscheel do you think it would be feasible with reasonable effort to allow specifying multiple AH finders here in the input file, without having to recompile? Something like this:

```yaml
ApparentHorizons:
  - Name: AhA
    InitialGuess:
      # ...
  - Name: AhB
    InitialGuess:
      # ...
```

I have been thinking that perhaps we can get this working by making the interpolation target an array component instead of a singleton, and instantiate on interpolation target for each AH finder. What do you think?

Reply:
Yes it would be nice to dynamically specify the number of apparent horizons. Not sure how much work it would be to make this switch. I'm guessing not trivial. We should have a discussion about this.

Reply:
I think we need to change the AH components from singletons to an array, then we can dynamically insert depending on the number requested. I think it would be nice to have this, but probably low priority. I also haven't thought about how this would affect the elements